.ci/ansible/files/user-data-centos.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+yum install -y python3
```

Comment:
I think this file is a left over for user_data. We can remove it

Reply:
Removed here: https://github.com/elastic/e2e-testing/pull/2034/commits/730c89ac4a847fcff19c5cdaa6a38ca11d7ea822